metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/enrichment_commands.py

```diff
@@ -190,7 +190,7 @@ def create_hbase_tables(self):
                   self.__params.hbase_principal_name,
                   execute_user=self.__params.hbase_user)
 
-        cmd = "echo \"create '{0}','{1}'\" | hbase shell -n"
+        cmd = "if [[ $(echo \"exists '{0}'\" | hbase shell | grep 'not exist') ]]; then echo \"create '{0}','{1}'\" | hbase shell -n; fi"
```

Comment:
Would it be possible to just add a `create_hbase_table` function in `metron_service` that is used by Enrichment, Indexing, and the Profiler?  Then we don't have to duplicate this in three places?

Reply:
I thought of that, the issue would be that that function would have the same parameters as this Execute command.  It wouldn't really make it better.

We could create a shared get_create_hbase_table_fmt() function that returns the command.

Reply:
That statement is based on us not always possibly wanting to use the same users etc.

Reply:
Still seems like not having to cut and paste that bit of shell script inside a python variable is worth generalizing to a `create_hbase_table` function.

Reply:
ok, i'll do that

Reply:
done